update infra release branch for release

sean-m-sullivan opened the pull request:
update infra release branch for release

github-actions[bot] committed:
Update pre-commit (#326)

Co-authored-by: branic <branic@users.noreply.github.com>

branic committed:
Update pre-commit

Tompage1994 committed:
Fix ansible-lint running against .py files (#332)

* Fix ansible-lint running against .py files

* Fix a pylint issue

rjo-uk committed:
Update description in the user role README (#333)

Updated the one line description in the role.

Co-authored-by: Sean Sullivan <ssulliva@redhat.com>

rochacbruno committed:
Fix version comparison (#337)

fix #334

alternative implementarion for #335

github-actions[bot] committed:
Update pre-commit (#338)

Co-authored-by: branic <branic@users.noreply.github.com>

dsavineau committed:
user: Fix password type field in README (#342)

The `password` type was set to bool instead of str.

Signed-off-by: Dimitri Savineau <dsavinea@redhat.com>

sean-m-sullivan committed:
update repository roles (#343)

Co-authored-by: David Danielsson <djdanielsson@users.noreply.github.com>

github-actions[bot] committed:
Update pre-commit (#344)

Co-authored-by: branic <branic@users.noreply.github.com>

dbk-rabel committed:
Fix wording to sync (#345)

branic committed:
Update pre-commit

Tompage1994 committed:
Fix markdown tables (#349)

Co-authored-by: Sean Sullivan <ssulliva@redhat.com>

github-actions[bot] committed:
Update pre-commit (#348)

Co-authored-by: branic <branic@users.noreply.github.com>
Co-authored-by: Sean Sullivan <ssulliva@redhat.com>

branic committed:
Update pre-commit

dbk-rabel committed:
collection_remote: set sync_dependencies default to PAH default (#346)

Co-authored-by: David Danielsson <djdanielsson@users.noreply.github.com>

m8ram committed:
Fix typo in ah_group_roles variable

Tompage1994 committed:
Add branch specifics to Contributing guide (#360)

Tompage1994 committed:
Fix documentation for validate_certs (#364)

Tompage1994 committed:
Fixed issue in all roles where AH_HOST was not being defaulted to if no variable set (#363)

Tompage1994 committed:
Fix documentation for group_roles role (#359)

github-actions[bot] committed:
Update pre-commit (#361)

Co-authored-by: branic <branic@users.noreply.github.com>

sean-m-sullivan committed:
Remove bindep (#367)

* Delete bindep.txt

Delete bindep as we only had python
https://github.com/ansible-collections/cisco.nxos does the same.

* Create remove_bindep.yml

* Update remove_bindep change to bugfix

branic committed:
Update pre-commit

djdanielsson commented on roles/collection_remote/tasks/main.yml:
```diff
@@ -27,7 +27,7 @@
     proxy_password:           "{{ __collection_remote_item.proxy_password | default(proxy_password | default(omit)) }}"
     state:                    "{{ __collection_remote_item.state | default(ah_state | default('present')) }}"
-    ah_host:                  "{{ ah_host | default(ah_hostname) }}"
+    ah_host:                  "{{ ah_host | default(ah_hostname) | default(omit) }}"
```
I might have missed it but what case would ah_host be omitted?

sean-m-sullivan replied:
Found it here, legit changed
https://github.com/ansible/galaxy_collection/pull/363

djdanielsson reviewed (changes requested):
I had one question that is repeated throughout

djdanielsson committed:
updating license and ansible min to match everywhere (#370)

* updating licnese and ansible min version

* updating runtime

* module missing license

Tompage1994 committed:
FIX: sanity failing due to added shebang in module_utils (#372)

Tompage1994 committed:
Fix CI stack (#373)

Tompage1994 committed:
Fix of Ci (#375)

* Upgrade docker-py

* downgradde docker

* trial new CI

* Try adding new ci cfg

* Update auth details

* fix reg port

* Remove initial seeding of container

* Update build_ignore

* FIX: sanity failing due to added shebang in module_utils (#372)

* trial new CI

Try adding new ci cfg

Update auth details

fix reg port

Remove initial seeding of container

Update build_ignore

* change where the collection is built

* change timing of collection build

* fix EOF

* control which roles are run

* FIX

* Trigger Git Commit

* Add group creation

* Re-order test playbooks
